reproducibility-with-guix.org

```diff
@@ -475,31 +497,37 @@ Now we are finally ready for the ultimate step in dependency analysis: identifyi
-        (unique-inputs (package-direct-inputs hello)))
-(format #t "Build inputs: ~a\n"
-        (unique-inputs
-         (bag-direct-inputs
-          (package->bag hello))))
+(format #t "Package inputs : ~a\n"
+        (length-and-list (unique-inputs (package-direct-inputs hello))))
+(format #t "Build inputs   : ~a\n"
+        (length-and-list
+         (unique-inputs
+          (bag-direct-inputs
+           (package->bag hello)))))
 (format #t "Package closure: ~a\n"
-        (delete-duplicates
-         (map package->specification
-              (package-closure (list hello)))))
+        (length-and-list
+         (delete-duplicates
+          (map package->specification
+               (package-closure (list hello))))))
 #+end_src
 
 #+RESULTS:
-: Package: hello@2.10
-: Package inputs: ()
-: Build inputs: ([source code from mirror://gnu/hello/hello-2.10.tar.gz] tar@1.32 gzip@1.10 bzip2@1.0.6 xz@5.2.4 file@5.33 diffutils@3.7 patch@2.7.6 findutils@4.6.0 gawk@5.0.1 sed@4.7 grep@3.3 coreutils@8.31 make@4.2.1 bash-minimal@5.0.7 ld-wrapper@0 binutils@2.32 gcc@7.4.0 glibc@2.29 glibc-utf8-locales@2.29)
-: Package closure: (m4@1.4.18 libatomic-ops@7.6.10 gmp@6.1.2 libgc@7.6.12 libltdl@2.4.6 libunistring@0.9.10 libffi@3.2.1 pkg-config@0.29.2 guile@2.2.6 libsigsegv@2.12 lzip@1.21 ed@1.15 perl@5.30.0 guile-bootstrap@2.0 zlib@1.2.11 xz@5.2.4 ncurses@6.1-20190609 libxml2@2.9.9 attr@2.4.48 gettext-minimal@0.20.1 gcc-cross-boot0-wrapped@7.4.0 libstdc++@7.4.0 ld-wrapper-boot3@0 bootstrap-binaries@0 ld-wrapper-boot0@0 flex@2.6.4 glibc-intermediate@2.29 libstdc++-boot0@4.9.4 expat@2.2.7 gcc-mesboot1-wrapper@4.7.4 mesboot-headers@0.19 gcc-core-mesboot@2.95.3 bootstrap-mes@0 bootstrap-mescc-tools@0.5.2 tcc-boot0@0.9.26-6.c004e9a mes-boot@0.19 tcc-boot@0.9.27 make-mesboot0@3.80 gcc-mesboot0@2.95.3 binutils-mesboot0@2.20.1a make-mesboot@3.82 diffutils-mesboot@2.7 gcc-mesboot1@4.7.4 glibc-headers-mesboot@2.16.0 glibc-mesboot0@2.2.5 binutils-mesboot@2.20.1a linux-libre-headers@4.19.56 linux-libre-headers-bootstrap@0 gcc-mesboot@4.9.4 glibc-mesboot@2.16.0 gcc-cross-boot0@7.4.0 bash-static@5.0.7 gettext-boot0@0.19.8.1 python-minimal@3.5.7 perl-boot0@5.30.0 texinfo@6.6 bison@3.4.1 gzip@1.10 libcap@2.27 acl@2.2.53 glibc-utf8-locales@2.29 gcc-mesboot-wrapper@4.9.4 file-boot0@5.33 findutils-boot0@4.6.0 diffutils-boot0@3.7 make-boot0@4.2.1 binutils-cross-boot0@2.32 glibc@2.29 gcc@7.4.0 binutils@2.32 ld-wrapper@0 bash-minimal@5.0.7 make@4.2.1 coreutils@8.31 grep@3.3 sed@4.7 gawk@5.0.1 findutils@4.6.0 patch@2.7.6 diffutils@3.7 file@5.33 bzip2@1.0.6 tar@1.32 hello@2.10)
+: Package        : hello@2.10
+: Package inputs : (0 ())
+: Build inputs   : (20 ([source code from mirror://gnu/hello/hello-2.10.tar.gz] tar@1.30 gzip@1.9 bzip2@1.0.6 xz@5.2.4 file@5.33 diffutils@3.6 patch@2.7.6 findutils@4.6.0 gawk@4.2.1 sed@4.5 grep@3.1 coreutils@8.30 make@4.2.1 bash-minimal@4.4.23 ld-wrapper@0 binutils@2.31.1 gcc@5.5.0 glibc@2.28 glibc-utf8-locales@2.28))
```

Comment:
Your Guix installation must be quite old, with gcc 5.5 and glibc 2.28!

Reply:
hum? maybe but I `guix pull` yesterday. Weird. I will investigate. Thank you for the remark.